sdks/python/apache_beam/runners/portability/local_job_service.py

```diff
@@ -259,6 +273,7 @@ def run(self):
   def _run_job(self):
     self.set_state(beam_job_api_pb2.JobState.RUNNING)
     with JobLogHandler(self._log_queues):
+      self._update_dependencies()
```

Comment:
It may be good to log what's going on during job execution? We do something like it in Dataflow, right?

Reply:
Yes, in fact the interactive team is looking at improving things here. This particular line is just updating a proto field.